lib/importer/nunjucks/importer/macros/table_view.njk

```diff
@@ -1,18 +1,20 @@
 
 
-{% macro importerTableView(data) %}
+{% macro importerTableView(data, hideHeader=false) %}
```

Comment:
Is this used?

Reply:
Yes, we don't want headers when selecting headers but we do when we know what the headers are.